src/http/api/routes/webui.js

```diff
@@ -1,17 +1,26 @@
 'use strict'
 
+const Joi = require('@hapi/joi')
 const resources = require('../../gateway/resources')
 
 module.exports = [
   {
     method: '*',
-    path: '/ipfs/{cid*}',
+    path: '/ipfs/{path*}',
```

Comment:
Does this need an `/ipns` handler also?

Reply:
No, this is only for `/webui` on API port.
(`/ipns/` is only available on the Gateway port)